content_sync/pipelines/definitions/concourse/theme_assets_pipeline_test.py

```diff
@@ -0,0 +1,80 @@
+import json
+from urllib.parse import quote
+
+import pytest
+
+from content_sync.constants import DEV_ENDPOINT_URL
+from content_sync.pipelines.definitions.concourse.common.identifiers import (
+    OCW_HUGO_THEMES_GIT_IDENTIFIER,
+)
+from content_sync.pipelines.definitions.concourse.theme_assets_pipeline import (
+    ThemeAssetsPipelineDefinition,
+)
+
+
+@pytest.mark.parametrize("is_dev", [True, False])
+def test_generate_theme_assets_pipeline_definition(mocker, is_dev):
+    """
+    The theme assets pipeline definition should contain the expected properties
+    """
+    mock_is_dev = mocker.patch(
+        "content_sync.pipelines.definitions.concourse.theme_assets_pipeline.is_dev"
+    )
+    mock_is_dev.return_value = is_dev
+    artifacts_bucket = "ol-eng-artifacts"
+    preview_bucket = "ocw-content-preview"
+    publish_bucket = "ocw-content-publish"
+    ocw_hugo_themes_branch = "main"
+    instance_vars = f"?vars={quote(json.dumps({'branch': ocw_hugo_themes_branch}))}"
+    pipeline_definition = ThemeAssetsPipelineDefinition(
+        artifacts_bucket=artifacts_bucket,
+        preview_bucket=preview_bucket,
+        publish_bucket=publish_bucket,
+        ocw_hugo_themes_branch=ocw_hugo_themes_branch,
+        instance_vars=instance_vars,
+    )
+    rendered_definition = json.loads(pipeline_definition.json(indent=2))
```

Comment:
I know you don't have any `TryStep`s in this pipeline but w.r.t. https://github.com/mitodl/ol-infrastructure/issues/1637 

I know you don't have any `TryStep`s in this pipeline but

If you add the argument `by_alias=True` to the call to `json()` it will render the `try_` as it should be: `try`.

Reply:
There actually are `TryStep`s in the `ClearCdnCacheStep` and `SlackAlertStep`, so this is useful. In testing I've just been calling `.replace("try_", "try")` but this is much better. Still, it would be good for the alias to be rendered by default in this case as you would never want `try_` to show up in the resulting pipeline definition.